packages/gatsby/src/utils/webpack.config.js

```diff
@@ -166,6 +166,7 @@ module.exports = async (
           // ids to reduce filesize.
           new webpack.NamedModulesPlugin(),
           new FriendlyErrorsWebpackPlugin({
+            clearConsole: false,
```

Comment:
💯 yeah I've heard/helped several people that got stuck because they never saw the errors